addons/l10n_nl/data/account.account.template.csv

```diff
@@ -6,8 +6,8 @@
 "0101","Aanschafwaarde Grond ","0101","account.data_account_type_fixed_assets","l10n_nl.l10nnl_chart_template","l10n_nl.account_tag_23","False"
 "0110","Aanschafwaarde Gebouwen","0110","account.data_account_type_fixed_assets","l10n_nl.l10nnl_chart_template","l10n_nl.account_tag_23","False"
 "0119","Afschrijving Gebouwen","0119","account.data_account_type_fixed_assets","l10n_nl.l10nnl_chart_template","l10n_nl.account_tag_23","False"
-"0120","Aanschafwaarde Winkels","0120","account.data_account_type_fixed_assets","l10n_nl.l10nnl_chart_template","l10n_nl.account_tag_23","False"
-"0129","Afschrijving Winkels","0129","account.data_account_type_fixed_assets","l10n_nl.l10nnl_chart_template","l10n_nl.account_tag_23","False"
+"0120","Aanschafwaarde Winkels","0120","account.data_account_type_fixed_assets","l10n_nl.l10nnl_chart_template","l10n_nl.account_tag_23","True"
+"0129","Afschrijving Winkels","0129","account.data_account_type_fixed_assets","l10n_nl.l10nnl_chart_template","l10n_nl.account_tag_23","True"
```

Comment:
The Netherlands are not Anglo-Saxon (check if bool is set True on chart template), so here we should not do this.

Reply:
@jco-odoo 
I have reverted those changes in Netherlands data.